graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -654,15 +656,15 @@ def _get_graphql_type(self, class_name, property_definition):
                                          .format(class_name, property_definition))
                 # Don't include the fields and implemented interfaces, this information is already
```

Comment:
I still strongly disagree with this.

Reply:
I agree with your argument why we shouldn't do this, but this is not the scope of this PR.